container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/logs_collector_functions/implementations/fluentbit/consts.go

```diff
@@ -34,33 +32,26 @@ const (
 	listen {{.Input.Listen}}
 	port {{.Input.Port}}
 	storage.type  {{.Input.StorageType}}
-[FILTER]
-	name {{.Filter.Name}}
-	match {{.Filter.Match}}
-	{{.Filter.GetRulesStr}}
 [OUTPUT]
 	name {{.Output.Name}}
 	match {{.Output.Match}}
 	host {{.Output.Host}}
 	port {{.Output.Port}}
-	labels {{.Output.GetLabelsStr}}
-	line_format {{.Output.LineFormat}}
-	tenant_id_key {{.Output.TenantIDKey}}
-	retry_limit {{.Output.RetryLimit}}
 `
 
 	healthCheckEndpointPath = "api/v1/health"
-	////////////////////////--FINISH LOKI CONTAINER CONFIGURATION SECTION--/////////////////////////////
+	////////////////////////--FINISH FLUENT BIT CONTAINER CONFIGURATION SECTION--/////////////////////////////
 
 	////////////////////////--FLUENTBIT CONFIGURATION SECTION--/////////////////////////////
```

Comment:
nit: it it expected that we now have 2 `FLUENTBIT CONFIGURATION SECTION` here?

Reply:
yes it is, one is for container specific configuration and the other is for the fluentbit service itself.